examples/double_mach_reflection_2d/double_mach_reflection_2d.ini

```diff
@@ -0,0 +1,87 @@
+[backend]
```

Comment:
Are you okay to remove the examples now that they'll be in their own test cases repo?